### PR TITLE
Ports supply pods from /tg/

### PR DESCRIPTION
**Pull request by DominikPanic**
This was pretty hard

### CODE REVIEW

**Comment by Rohesie** [comment]
Yeeeeesssss

File: code/__HELPERS/icons.dm [hunk 1]
```diff
@@ -1167,4 +1167,14 @@ world
...
 	else
-		return humanoid_icon_cache[icon_id]
\ No newline at end of file
+		return humanoid_icon_cache[icon_id]
+
+
+/proc/Get_Pixel_Angle(y, x)
+	if(!y)
+		return (x >= 0) ? 90 : 270
+	. = arctan(x / y)
```
Comment (spookydonut) [inline code/__HELPERS/icons.dm]: beware byond trig functions have errors, see shuttle code for less buggy angle calc

Comment (DominikPanic) [reply code/__HELPERS/icons.dm]: Which one should I use?

Comment (DominikPanic) [reply code/__HELPERS/icons.dm]: Also this is just for a visual effect so any errors don't matter that much

**Pull request merged**